src/syncerd/types.rs

```diff
@@ -305,10 +305,11 @@ pub struct FeeEstimation {
 
 #[derive(Clone, Debug, Display, StrictEncode, StrictDecode, Eq, PartialEq, Hash)]
 #[display(Debug)]
+// the sats per kvB is because we need u64 for Eq, PartialEq and Hash
```

Comment:
Thank you for the comments and I like the increased precission we get from calculating per kvbyte instead of vbyte.